Helper/Webhook.php

```diff
@@ -109,7 +81,7 @@ public function __construct(
         $this->logger = $logger;
         $this->orderHelper = $orderHelper;
         $this->orderRepository = $orderRepository;
-        self::$webhookHandlerFactory = $webhookHandlerFactory;
+        $this->webhookHandlerFactory = $webhookHandlerFactory;
```

Comment:
Good change, I don't see any valid reason why this variable was static at the first place :)